ote/src/cljs/ote/views/transport_service_common.cljs

```diff
@@ -66,84 +66,100 @@
-     [:div
-      [:p (tr [:form-help :external-interfaces-end])]])
-
-    {:name ::t-service/notice-external-interfaces?
-     :type :checkbox
-     :required? true
-     :style style-form/padding-top
-     :validate [[:checked?]]}))
+    (form/group
+      {:label (tr [:field-labels :transport-service-common ::t-service/external-interfaces])
+       :columns 3}
+
+      (form/info
+        [:div
+         [:div {:style {:margin-bottom "5px"}}
+          [:b (if (= :schedule sub-type)
+                [:span (str (tr [:form-help :external-interfaces-intro-rae]) " ")
+                 [linkify "https://liikennevirasto.fi/rae" (tr [:form-help :RAE-link-text])
+                  {:target "_blank"}]]
+                (tr [:form-help :external-interfaces-intro]))]]
+         [:div (tr [:form-help :external-interfaces])]
+         [dialog
+          (tr [:form-help :external-interfaces-read-more :link])
+          (tr [:form-help :external-interfaces-read-more :dialog-title])
+          [:div
+           (tr [:form-help :external-interfaces-read-more :dialog-text])]]
+         (when (= :passenger-transportation type)
+           [:div {:style {:margin-top "20px"}}
+            [:b (tr [:form-help :external-interfaces-payment-systems])]])])
+
+      {:name ::t-service/external-interfaces
+       :type :table
+       :prepare-for-save values/without-empty-rows
+       :table-fields [{:name ::t-service/data-content
+                       :type :multiselect-selection
+                       :tooltip (tr [:form-help :external-interfaces-tooltips :data-content])
+                       :width "20%"
+                       :auto-width? true
+                       :full-width? true
+                       :options t-service/interface-data-contents
+                       :show-option (tr-key [:enums ::t-service/interface-data-content])
+                       :required? true
+                       :is-empty? validation/empty-enum-dropdown?}
+                      {:name ::t-service/external-service-url
+                       :type :string
+                       :tooltip (tr [:form-help :external-interfaces-tooltips :external-service-url])
+                       :width "20%"
+                       :full-width? true
+                       :on-blur #(e! (ts/->EnsureExternalInterfaceUrl (-> % .-target .-value)))
+                       :read (comp ::t-service/url ::t-service/external-interface)
+                       :write #(assoc-in %1 [::t-service/external-interface ::t-service/url] %2)
+                       :required? true}
+                      {:name :ext-validation
+                       :type :component
+                       :component (fn [{{status :status} :data}]
+                                    (if-not status
+                                      [:span]
+                                      (if (= :success status)
+                                        [(tooltip-wrapper ic/action-done) {:style {:width 24 :height 24
```

Comment:
Consider creating new ui tooltip style file where 
You have base style for this action button.
like:
(defn base-style {:width 24 :height 24 :vertical-align "middle"}

And then only merge style map here.
->
(merge {:color "green"} <and the base style filename + function name here<)